app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt

```diff
@@ -174,3 +175,13 @@ fun isWeekend(i: Int, isSundayFirst: Boolean): Boolean {
         i == 5 || i == 6 || i == 12 || i == 13
     }
 }
+
+fun getDateFormatsWithYear() = arrayListOf(
```

Comment:
where did you get these formats from? You could reuse the ones from https://github.com/SimpleMobileTools/Simple-Commons/blob/master/commons/src/main/kotlin/com/simplemobiletools/commons/helpers/Constants.kt#L286 , I haven't seen other ones so far. Or maybe add the extra ones in Simple Commons too, if it makes sense.

Reply:
Actually got them from [there](https://github.com/SimpleMobileTools/Simple-Commons/blob/master/commons/src/main/kotlin/com/simplemobiletools/commons/helpers/Constants.kt#L286 ). I would add them